models/intel/person-detection-0100/description/person-detection-0100.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0100
```

Comment:
How do these models correlate with `pedestrian-detection-*` family?

Reply:
if I right understand they should correlate with person-detection-retail family, not pedestrian (difference in use case)

Reply:
not all persons are pedestrians